Readme.md

```diff
@@ -1,3 +1,9 @@
+[![Gem Version](https://badge.fury.io/rb/ar_after_transaction.png)](http://badge.fury.io/rb/ar_after_transaction)
+[![Travis Build Status](https://secure.travis-ci.org/evolve75/RubyTree.png)](http://travis-ci.org/grosser/ar_after_transaction)
+[![Dependency Status](https://gemnasium.com/evolve75/ar_after_transaction.png)](https://gemnasium.com/grosser/ar_after_transaction)
+[![Code Climate](https://codeclimate.com/github/evolve75/RubyTree.png)](https://codeclimate.com/github/grosser/ar_after_transaction)
+[![Coverage Status](https://coveralls.io/repos/evolve75/RubyTree/badge.png)](https://coveralls.io/r/grosser/ar_after_transaction)
```

Comment:
I think it will only work if you turn it on from your side, as the owner of the repo. It is free. It can be a fun brag, and reassuring to people who are considering starting to use the gem. It is nothing that one couldn't get from running simplecov locally or looking through the specs with a discerning eye.